.github/scripts/markdown-link-check-config.json

```diff
@@ -4,5 +4,6 @@
       "pattern": "^https://mvnrepository\\.com/artifact/"
     }
   ],
-  "retryOn429": true
+  "retryOn429": true,
+  "aliveStatusCodes": [ 200, 403 ]
```

Comment:
this might deserve a comment, since it definitely looks funny

Reply:
yes, but where to put the comment (damn you json!)

Reply:
oh. yeah. :(